.gitlab/build-deb-rpm-snapshot.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#This script uses verdaccio to publish the package to a local registry
+#We build the npm package setting the snapshot version and then we publish it to the local registry
+#Use local registry  to build the deb and rpm packages
+
+verdaccio &
+sleep 5
+npm install -g npm-cli-adduser
+npm-cli-adduser -u test -p test -e email@email.com -r http://localhost:4873
+yarn install
+content=`cat ./package.json | tr '\n' ' '`
+current_version=$(jq '.version' <<< "$content" )
+current_version=$(echo "$current_version" | tr -d '"')
+current_version+="$CI_VERSION_SUFFIX"
+npm version --no-git-tag-version $current_version
+npm publish --tag dev --registry http://localhost:4873
```

Comment:
`npm pack` generates the .tgz locally

```
...
npm notice === Tarball Details === 
npm notice name:          dd-trace                                
npm notice version:       6.0.0-pre                               
npm notice filename:      dd-trace-6.0.0-pre.tgz                  
npm notice package size:  380.9 kB                                
npm notice unpacked size: 1.7 MB                                  
npm notice shasum:        2587cb506524badfa8db1f77cfdb29c8f771cc9b
npm notice integrity:     sha512-vHPbq8jGEHwKR[...]QISCMIPIiA4Ew==
npm notice total files:   519                                     
npm notice 
dd-trace-6.0.0-pre.tgz
```

if `fpm` allows to generate the package from a path to .tgz, could you avoid to use verdaccio?

Reply:
Before generating the gitlab image, I tried to do what you suggest, but I couldn't get it to work.
To follow the path you propose, I would need some help from the nodejs team.